src/proper_target.erl

```diff
@@ -258,16 +281,17 @@ init([{Strategy, Data}]) ->
 %% @private
 -spec handle_call(Request :: term(), From :: {pid(), Tag :: term()},
                   State :: state()) ->
-  {reply, Reply :: term(), NewState :: state()}.
-handle_call(gen, _From, State) ->
-  Strat = State#state.strategy,
-  Target = State#state.target,
-  Data = State#state.data,
+        {reply, Reply :: term(), NewState :: state()}.
+handle_call(gen, _From,
```

Comment:
I think this looks better as:
```erlang
handle_call(gen, _From, State) ->
  #state{strategy = Strat, target = Target, data = Data} = State,
  ...
```
because it's immediately clear that there no pattern matching on the State variable.